index.bs

```diff
@@ -889,6 +792,102 @@ user consent to a specific transaction. The structure of these signatures is def
+
+        Its value's {{ScopedCredentialEntity/name}} member is required, and contains a friendly name for the user account (e.g.
+        "john.p.smith@example.com", or "John P. Smith").
+
+        Its value's {{ScopedCredentialEntity/id}} member is required, and contains an identifier for the account, specified by
+        the [=relying party=]. This is not meant to be displayed to the user, but is used by the relying party to control the
+        number of credentials - an authenticator will never contain more than one credential for a given relying party under the
+        same {{ScopedCredentialEntity/id}}.
+
+    :   <dfn>challenge</dfn>
+    ::  This member contains a channelge intended to be used for generating the newly created credential's [=attestation object=].
+
+    :   <dfn>parameters</dfn>
+    ::  This member contains information about the desired properties of the credential to be created. The sequence is ordered
+        from most preferred to least preferred. The platform makes a best-effort to create the most preferred credential that it
+        can.
+
+    :   <dfn>timeout</dfn>
+    ::  This member specifies a time, in milliseconds, that the caller is willing to wait for the call to complete. This is
+        treated as a hint, and may be overridden by the platform.
+
+    :   <dfn>exclude</dfn>
+    ::  This member is intended for use by [=[RPS]=] that wish to limit the creation of multiple credentials for the same
+        account on a single authenticator. The platform is requested to return an error if the new credential would be created
+        on an authenticator that also contains one of the credentials enumerated in this parameter.
+
+    :   <dfn>attachment</dfn>
+    ::  This member contains authenticator attachment descriptions, which are used as an additional constraint on which
+        authenticators are eligible to participate in a [[#makeCredential]] or [[#getAssertion]] operation. See [[#attachment]]
+        for a description of the attachment values and their meanings.
+
+    :   <dfn>extensions</dfn>
+    ::  This member contains additional parameters requesting additional processing by the client and authenticator. For example,
+        the caller may request that only authenticators with certain capabilies be used to create the credential, or that
+        particular information be returned in the [=attestation object=]. Some extensions are defined in [[#extensions]];
+        consult the IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registries]] for an up-to-date list
+        of registered WebAuthn Extensions.
+</div>
+
+### Entity Description ### {#dictionary-scopedcredentialentity}
+
+The {{ScopedCredentialEntity}} dictionary describes a user account or a [=relying party=] with which a credential is associated.
+
+<xmp class="idl">
+    dictionary ScopedCredentialEntity {
+        DOMString id;
```

Comment:
+1 to @leshi's question "In `RelyingPartyUserInfo` some of these were required, why the change?"